src/components/file/examples/file-accepted-types.tsx

```diff
@@ -0,0 +1,40 @@
+import { FileInfo } from '@limetech/lime-elements';
```

Comment:
I added `fix: #1174` to the commit message and the PR description. That way, when the fix is released, the changelog will contain links to the commit and the issue, like [this](https://github.com/Lundalogik/lime-elements/releases/tag/v30.15.2).